### PR TITLE
Add neural speed example

### PR DESCRIPTION
**Pull request by grorge123**

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by grorge123** [comment]
I run a simple test on i7-12700K.
model: llama-2-7b-chat.Q4_0
Number of input tokens: 12

- llama.cpp:
    Number of output tokens: 170
    load time =    2706.11 ms
    sample time =       3.59 ms /   170 runs   (    0.02 ms per token, 47393.36 tokens per second)
    prompt eval time =     478.58 ms /    12 tokens (   39.88 ms per token,    25.07 tokens per second)
    eval time =   34458.22 ms /   169 runs   (  203.89 ms per token,     4.90 tokens per second)
    total time =   37193.88 ms /   181 tokens
- Neural Speed:
    Number of Output tokens: 409
    Load time: 5290 ms
    Compute time: 157988 ms
    Total time: 163278 ms

**Comment by juntao** [comment]
Does that mean it is actually slower than llama.cpp?

**Comment by grorge123** [comment]
Yes, the current result shows neural speed is slower than llama.cpp. In addition, directly run neural speed also can not achieve llama.cpp runtime.

**Comment by hydai** [comment]
@grorge123 
Could you check how many CPU cores are used in both benchmarks? According to the document, the neural speed should have a better performance on the Intel CPU than other runtimes.

**Comment by grorge123** [comment]
Neural speed uses all the CPU cores(20). I update the neural speed vision to 1.0.
The new result is
Load time: 4989ms
Compute time: 96263ms

I test on another computer i7-10700. The other variables are the same.
- llama.cpp:
Number of output tokens: 512
load time =    1094.42 ms
sample time =      13.92 ms /   512 runs   (    0.03 ms per token, 36773.68 tokens per second)
prompt eval time =     675.74 ms /    12 tokens (   56.31 ms per token,    17.76 tokens per second)
eval time =   77484.50 ms /   511 runs   (  151.63 ms per token,     6.59 tokens per second)
total time =   78824.58 ms /   523 tokens
- neural speed
Number of Output tokens: 409
Load time: 7923ms
Compute time: 59378ms

In this case neural speed has a better performance than llama.cpp. But I have no idea why i7-10700 has better performance than i7-12700K.

File: wasmedge-neuralspeed/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+cd <path/to/your/wasmedge/source/folder>
+
+cmake -GNinja -Bbuild -DCMAKE_BUILD_TYPE=Release -DWASMEDGE_PLUGIN_WASI_NN_BACKEND="neuralspeed"
+cmake --build build
+
+# For the WASI-NN plugin, you should install this project.
+cmake --install build
+```
+
+Then you will have an executable `wasmedge` runtime under `/usr/local/bin` and the WASI-NN with OpenVINO backend plug-in under `/usr/local/lib/wasmedge/libwasmedgePluginWasiNN.so` after installation.
```
Comment (dm4) [inline wasmedge-neuralspeed/README.md:29]: Should be `NeuraSpped` backend?

**Review by dm4 — CHANGES_REQUESTED** [review]

File: wasmedge-neuralspeed/README.md (hunk 1 above)
Comment (grorge123) [reply wasmedge-neuralspeed/README.md]: I have fixed it. Please check, thx.

**Pull request merged**